docs/docs/how-to/windows-development-setup.md

```diff
@@ -51,6 +55,5 @@ Next, you should start the amazing [**Redwood Tutorial**](tutorial/chapter1/inst
 
 Or run `yarn create redwood-app myApp` to get started with a new project.
 
-
 >⚠️ Heads Up
```

Comment:
We can replace the Heads-Up for a troubleshooting section. 
Adding your error and how to solve it.

Reply:
"Usually, Microsoft Visual C++ Redistributable is installed in with Node.js as a dependency": [Ajit Goel]: I has installed NodeJs on my machine(which did not install Microsoft Visual C++ Redistributable) when I got the above error. 
"This issue of Prisma seem to be an edge case when you work with Windows on ARM64, is it your case ?" : [Ajit Goel] I have a 64 bit windows 10 Dell latitude machine. Not sure if it is ARM64 or not.
"Otherwise, I think it will be good if we have a troubleshooting section with context and how to solve it." [Ajit Goel] I strongly agree.